binding.c

```diff
@@ -176,7 +176,7 @@ on_udx_stream_read (udx_stream_t *stream, ssize_t read_len, const uv_buf_t *buf)
   n->read_buf_head += buf->len;
   n->read_buf_free -= buf->len;
 
-  if (n->mode == UDX_NAPI_NON_INTERACTIVE && n->read_buf_free >= UDX_MTU) {
+  if (n->mode == UDX_NAPI_NON_INTERACTIVE && n->read_buf_free >= stream->mtu) {
```

Comment:
this is actually the remote mtu, but we don't know it (nor should we), so it's prob safer here to do something like 2 * UDX_MTU